ee/frontend/mobile-replay/mobile.types.ts

```diff
@@ -122,6 +122,60 @@ type wireframeBase = {
     style?: MobileStyles
 }
 
+export type wireframeInputBase = wireframeBase & {
+    type: 'input'
+    disabled: boolean
+}
+
+export type wireframeCheckBox = wireframeInputBase & {
+    inputType: 'checkbox'
+    checked: boolean
+    label: string
+}
+
+export type wireframeRadioGroup = wireframeBase & {
+    groupName: string
+}
+
+export type wireframeRadio = wireframeInputBase & {
+    inputType: 'radio'
+    checked: boolean
+    label: string
+}
+
+export type wireframeInput = wireframeInputBase & {
+    inputType: 'text' | 'password' | 'email' | 'number' | 'search' | 'tel' | 'url'
+    value: string
+}
+
+export type wireframeSelect = wireframeInputBase & {
+    inputType: 'select'
+    value: string
+    options: string[]
```

Comment:
```suggestion
    options?: string[]
```

Usually has no items while its preloading